assets/queries/cloudFormation/iot_policy_allows_action/test/positive.yaml

```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: A sample template
+Resources:
+    IoTPolicy:
+      Type: AWS::IoT::Policy
+      Properties:
+        PolicyDocument:
+          Version: '2012-10-17'
+          Statement:
+          - Effect: Allow
+            Action: "*"
+            Resource:
+            - arn:aws:iot:us-east-1:123456789012:client/client
+          - Effect: Deny
+            Action:
+            - sqs:*
+            NotResource:
+            - !GetAtt myqueue.Arn
```

Comment:
Resource `myqueue` does not exist in this template. Remove the `GetAtt `function and leave the ARN hardcoded:

`NotResource: my-hardcoded-arn`